documentation/catalog/guides/adding_a_new_provider.md

```diff
@@ -128,12 +130,34 @@ PROVIDER_WORKFLOWS = [
 There are many other options that allow you to tweak the `schedule` (when and
 how often your DAG is run), timeouts for individual steps of the DAG, and more.
 These are documented in the definition of the `ProviderWorkflow` dataclass.
-_<TODO: add docs for other options.>_
+
+<!--TODO: add docs for other options.-->
 
 After adding your configuration, run `just up` and you should now have a fully
-functioning provider DAG! _<TODO: add and link to docs for how to run provider
-DAGs locally, preferably with images.>_
+functioning provider DAG!
+
+<!--TODO: add and link to docs for how to run provider DAGs locally, preferably with images.-->
 
 _NOTE_: when your code is merged, the DAG will become available in production
 but will be disabled by default. A contributor with Airflow access will need to
 manually turn the DAG on in production.
+
+## Testing guide
+
+### Steps
+
+1. Ensure you've gone through the [quickstart](/catalog/guides/quickstart.md).
+   Ensure that the Docker daemon is running.
+
+2. Run individual test.
+
+   ```console
+   $ just catalog/test-session
+   $ pytest -k <provider_name>
+   ```
+
+### Alternative
+
+1. ```console
+   $ just catalog/test -k <provider_name>
```

Comment:
This formatting should allow this to still be under point number 2: 

```suggestion
   Alternatively, the test selection can be run in Docker directly with:

   ```console
   $ just catalog/test -k <provider_name>
```

![image](https://github.com/WordPress/openverse/assets/10214785/abf403f4-8094-4649-bd9f-7b647959b4e9)

Reply:
Thank you @AetherUnbound for the suggestion.